nixos/tests/all-tests.nix

```
@@ -1094,6 +1094,7 @@ in
   public-inbox = handleTest ./public-inbox.nix { };
   pufferpanel = handleTest ./pufferpanel.nix { };
   pulseaudio = discoverTests (import ./pulseaudio.nix);
+  qbittorrent = handleTest ./qbittorrent.nix { };
```

Can you use runTest instead ?

---

```suggestion
  qbittorrent = runTest ./qbittorrent.nix;
```

Would this be enough?

---

No, please have a look at other tests using `runTest`, see the changes in the test file. This is mostly the way the test is declared.

Look at this PR, it might help as well: https://github.com/NixOS/nixpkgs/issues/386873

---

Seems like you know how to do it, could you do suggest the required changes? I don't want to learn nixos test system just yet.

---

As I mentioned and to reduce the ball tossing, have look at the mentioned PR in my previous comment, you'll find plenty of examples much more self-explanatory than any other explanation.

---

Please note I'm not the author of this PR. Good pointers for whoever will be address it though.

---

Btw, it kinda looks like it would be enough - see https://github.com/fsnkty/nixpkgs/blob/ae375e942eef10c579ac274e503da4dc50d9629c/nixos/tests/all-tests.nix#L170

---

Good point. I wrote the test initially using a somewhat dated guide. Should be fixed now: https://github.com/fsnkty/nixpkgs/pull/3/commits/a04cfe28f732ad87cef7ce34ef6f6d486f5d64e8. 

Aside from removing the `handleTest` part it was also necessary to remove the `import ./make-test-python.nix ( )` part.